components/lists/Signaling.tsx

```diff
@@ -1,18 +1,19 @@
 import { forwardRef } from "react"
-import { View, StyleSheet } from "react-native"
+import { View, StyleSheet, FlatList } from "react-native"
 import CustomText from "@/components/ui/CustomText"
 import BottomDrawer from "@/components/BottomDrawer"
 import { CustomButton } from "@/components/ui/Button"
-import { BottomSheetSectionList } from "@gorhom/bottom-sheet"
```

Comment:
since the BottomSheet was removed, can you uninstall `@gorhom/bottom-sheet` too?

Reply:
i can't see this